front/src/modules/SimulationReportSheet/Route.tsx

```diff
@@ -0,0 +1,124 @@
+import type { StdcmSuccessResponse } from 'applications/stdcm/types';
+
+import styles from './styles/SimulationReportStyleSheet';
+import formatRouteTable from './utils/formatRouteTable';
+
+type RouteProps =
+  | { mode: 'stdcm'; stdcmData: StdcmSuccessResponse }
+  | { mode: 'operationalStudies'; operationalPointsList: OperationalPointWithTimeAndSpeed[] };
+
+const RouteTable = (props: RouteProps) => {
+  const { t } = useTranslation('stdcm');
+  const { mode } = props;
+  const isStdcm = mode === 'stdcm';
+
+  const rows = formatRouteTable({ ...props });
+
+  return (
+    <View style={styles.consistAndRoute.stopTableContainer}>
+      <Table style={styles.consistAndRoute.stopTable}>
+        <TH style={styles.consistAndRoute.stopTableTH}>
+          <View style={styles.consistAndRoute.stopTableIndexWidth}>
+            <TD aria-label="line-count" />
+          </View>
+          <View style={styles.consistAndRoute.stopTableOpWidth}>
+            <TD>{t('reportSheet.operationalPoint')}</TD>
+          </View>
+          <View style={styles.consistAndRoute.stopTableChWidth}>
+            <TD>{t('reportSheet.code')}</TD>
+          </View>
+          <View style={styles.consistAndRoute.stopTableEndWidth}>
+            <TD>{t('reportSheet.endStop')}</TD>
+          </View>
+          {isStdcm && (
+            <View style={styles.consistAndRoute.stopForWidth}>
+              <TD>{t('reportSheet.stopTime')}</TD>
+            </View>
+          )}
+          <View style={styles.consistAndRoute.stopTableStartWidth}>
+            <TD>{t('reportSheet.startStop')}</TD>
+          </View>
+          <View style={styles.consistAndRoute.stopTableStopTypeWidth}>
+            {isStdcm && <TD>{t('reportSheet.stopType')}</TD>}
+          </View>
+        </TH>
+
+        {rows.map((row, idx) => {
+          const isLast = idx === rows.length - 1;
+          const isFirst = idx === 0;
+          return (
+            <TR key={row.index} style={styles.consistAndRoute.stopTableTbody}>
+              <View style={styles.consistAndRoute.stopTableIndexWidth}>
+                <TD style={styles.consistAndRoute.stopTableIndexColumn}>{row.index}</TD>
+              </View>
+              <View style={styles.consistAndRoute.stopTableOpWidth}>
+                <TD style={styles.consistAndRoute.stopTableOpColumn}>{row.name}</TD>
+              </View>
+              <View style={styles.consistAndRoute.stopTableChWidth}>
+                <TD style={styles.consistAndRoute.stopTableChColumn}>{row.secondaryCode}</TD>
+              </View>
+              <View style={styles.consistAndRoute.stopTableEndWidth}>
+                <TD
+                  style={
+                    row.italic
+                      ? styles.consistAndRoute.stopTableItalicColumn
+                      : styles.consistAndRoute.stopTableStartColumn
+                  }
+                >
+                  {row.arrivesAt}
+                  {row.tolerances && isLast && (
```

Comment:
You don't need the isLast, because if the row is not the last one or the first one, it will not have tolerance :+1: 

You can simply check if the row has tolerances or not
```suggestion
                  {row.tolerances && (
```

Same for isFirst below

Reply:
Done ! but I need to keep `&& row.arrivesAt` or `&& row.row.leavesAt`, otherwise tolerances are displayed in both columns